### ISSUE TITLE
Referendum pass threshold UI

### ISSUE THREAD

**Issue by niklasp** [Proof-Of-Chaos/website#9]
display on the ui the pass threshold

**Comment by niklasp** [comment]
@tumispro is the backend logic ready?

**Comment by gabrieljaegerde** [comment]
Is this just regarding the indicator whether a referndum is currently passing/failing?

**Comment by niklasp** [comment]
yeah like a line to see in the green and red bar whats the percentage important for passing / failing
